src/Microsoft.VisualStudio.Editors/PropPages/AdvBuildSettingsPropPage.vb

```diff
@@ -71,23 +68,8 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
         ''' </summary>
         Private Sub PopulateLanguageVersions()
             cboLanguageVersion.Items.Clear()
-
-            cboLanguageVersion.Items.AddRange(CSharpLanguageVersionUtilities.GetAllLanguageVersions())
```

Comment:
Can CSharpLanguageVersionUtilities be deleted if this is the only usage?

Reply:
I'll take a look.

Reply:
Yes, deleted.